source/simulator/parameters.cc

```diff
@@ -594,6 +594,18 @@ namespace aspect
     }
     prm.leave_subsection();
 
+    prm.enter_subsection ("Postprocess");
+    {
+      prm.declare_entry ("Run postprocessors on nonlinear iterations", "false",
+                         Patterns::Bool (),
+                         "Whether or not the postproccessors should be executed after "
+                         "each of the nonlinear iterations done within one time step. "
+                         "As this is mainly an option for the purposes of debugging, "
+                         "it is not supported when the 'Time between graphical output' "
```

Comment:
Maybe mention that 'Time between graphical output' is not available in all postprocessors. Something like:

As this is mainly an option for the purposes of debugging, "
"it is not supported in all postprocessors, e.g. when the 'Time between 
graphical output is larger than zero, or when the postprocessor is not intended to be run more than once per timestep."